MonoZelda/Link/Projectiles/ProjectileManager.cs

```diff
@@ -85,6 +87,9 @@ public void useSwordBeam(Player player)
     public void fireEquippedProjectile(Player player)
     {
         itemFired = projectile.GetProjectileObject(equippedProjectile, player);
+        if (itemFired is WoodenSword) {
+            return;
+        }
```

Comment:
Will need to be refactored when/if we introduce more sword types, but this is good for now